# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/hyprland/sane-defaults.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- hyprland-source/src/main.cpp.vanilla	2024-11-23 14:38:41.983271823 +0100
+++ hyprland-source/src/main.cpp	2024-11-23 14:43:51.793266787 +0100
@@ -49,7 +49,7 @@
     std::string              configPath;
     std::string              socketName;
     int                      socketFd   = -1;
-    bool                     ignoreSudo = false;
+    bool                     ignoreSudo = true;
 
     std::vector<std::string> args{argv + 1, argv + argc};
 
--- hyprland-source/src/config/defaultConfig.hpp.vanilla	2024-11-23 14:30:19.433279957 +0100
+++ hyprland-source/src/config/defaultConfig.hpp	2024-11-23 14:31:57.139945039 +0100
@@ -9,7 +9,7 @@
 # OR EDIT THIS ONE ACCORDING TO THE WIKI INSTRUCTIONS.
 # #######################################################################################
 
-autogenerated = 1 # remove this line to remove the warning
+autogenerated = 0 # remove this line to remove the warning
 
 # This is an example Hyprland config file.
 # Refer to the wiki for more information.
